btr-api/migrations/versions/20241007_1412_4aaab6597068_.py

```diff
@@ -0,0 +1,77 @@
+"""empty message
+
+Revision ID: 4aaab6597068
+Revises: 8d8279a86def
+Create Date: 2024-10-07 14:12:40.894440
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = '4aaab6597068'
+down_revision = '8d8279a86def'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.create_table('request',
+    sa.Column('id', sa.Uuid(), nullable=False),
+    sa.Column('full_name', sa.String(), nullable=False),
+    sa.Column('email', sa.String(), nullable=False),
+    sa.Column('birthdate', sa.Date(), nullable=False),
+    sa.Column('business_identifier', sa.String(), nullable=False),
+    sa.Column('information_to_omit', sa.Enum('ALL', 'FULL_NAME', 'BIRTH_YEAR', 'CITIZENSHIP_PR', name='informationtoomittype'), nullable=False),
+    sa.Column('individual_at_risk', sa.Enum('SI', 'HOUSEHOLD', name='individualatrisk'), nullable=False),
+    sa.Column('reasons', sa.String(), nullable=False),
+    sa.Column('completing_party', sa.Enum('SI', 'REPRESENTATIVE', name='completingparty'), nullable=False),
+    sa.Column('completing_name', sa.String(), nullable=False),
+    sa.Column('completing_email', sa.String(), nullable=False),
+    sa.Column('created_at', sa.DateTime(), nullable=False),
+    sa.Column('updated_at', sa.DateTime(), nullable=False),
+    sa.Column('version', sa.Integer(), nullable=False),
+    sa.PrimaryKeyConstraint('id'),
+    sqlite_autoincrement=True
```

Comment:
How does this work ? Which field is autoincrement, primary key ?
It might not be bad to have id as autoincrement, and have uuid as another field